script/Ciel/Day 11/11_01_CIEL11_1.txt

```diff
@@ -120,142 +121,144 @@ No, that's not right.
 [sha:fd7d1170d7f2c482b7b9c4ca7c77c0cbac77c080]{
 -- Page 5, Offset 28994.
 -- この両手は、倒れている子供の血で赤いだけ。
--- TRANSLATION HERE
+My hands are red from the collapsed child's blood.
 }
 [sha:5cf5b2cd7a28e578dbc25eca0ca4fc76251ff4c0]{
 -- Page 6, Offset 28995.
 -- つまりオレは。
--- TRANSLATION HERE
+Which means...
 }
 [sha:da4b36ab2c20a43e5df90ee04874cd37ed10b9c0]{
 -- Page 6, Offset 28996.
 -- ガキの時分から、人でなしの人殺しだった。
--- TRANSLATION HERE
+I've been a despicable murderer since I was a kid.
 }
 [sha:0be0a14ad349bdf20f2438ac0c117024e0546167]{
 -- Page 7, Offset 28997.
 -- 日の当たらぬ庭園。
--- TRANSLATION HERE
+It is a garden which the sun does not reach.
 }
 [sha:f59b093ad8d5cc2754b1f2aea1d4d9c29a1cd42f]{
 -- Page 7, Offset 28998.
 -- 月光だけを頼りに咲き誇る大輪の花々。
--- TRANSLATION HERE
+Where full flowers bloom under the guidance of nothing but moonlight.
```

Comment:
"Where full flowers bloom, relying on nothing but moonlight."

Reply:
That's probably more accurate, but man is my version cool